### ISSUE TITLE
Trouble with dump-gnash

### ISSUE THREAD

**Issue by apolzon** [wakoopa/Thwomp#1]
I realize this isn't truly related to your gem, but any suggestions you could provide would be most appreciated.

Following your instructions, I'm able to build gnash, but the dump-gnash gui executable is never created. If I add sdl to the --enable-gui list, it does get created tho. Did you run into anything like this during your gnash install?

Thanks.

**Comment by menno** [comment]
Which OS are you trying to install it on? 

On Ubuntu we're actually just using the `gnash` package in production. But on my Yosemite machine I have not tried to build it yet, but that has always been very complicated as well.

**Comment by apolzon** [comment]
Sorry, yes, I'm building on OSX Yosemite. FWIW I also had to manually add the boost_system lib in the configure step with LIBS=-lboost_system-mt.

What version of osx were you using when you setup the instructions in the readme?

**Comment by menno** [comment]
It seems the original instructions were based on OS X Lion. 

I tried to rebuild gnash on Yosemite and also ran into all sorts of trouble. I did not figure out all the problems within the time I had.

Did you make any progress?

**Comment by apolzon** [comment]
No, I never figured out why the dump gui wasn't getting built. Had to move on to other things since I have it working on my app servers.
